Hush/Assets/Scripts/Game/DoorManager.cs

```diff
@@ -0,0 +1,45 @@
+using UnityEngine;
+
+public class DoorManager : MonoBehaviour
+{
+    private bool isDoorOpen; // TODO: modify this value when the door operning conditionals are met
+    [SerializeField] private GameObject TextUI;
+    private Animator TextDoorAnimator;
+
+    void Start()
+    {
+        TextDoorAnimator = gameObject.GetComponentInChildren<Animator>();
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.gameObject.tag == "Player")
+        {
+            bool engagedInCombat = other.gameObject.GetComponent<PlayerInputManager>().isInCombat;
+            bool hasRelic = other.gameObject.GetComponent<PlayerPossessions>().possessesRelic;
+
+            if (!engagedInCombat && hasRelic)
+            {
```

Comment:
Could be nice to use input to trigger this and walk through (would be another task)